src/ssl.c

```diff
@@ -26074,39 +26074,39 @@ int wolfSSL_sk_CIPHER_description(WOLFSSL_CIPHER* cipher)
     /* Build up the string by copying onto the end. */
     XSTRNCPY(dp, name, len);
     dp[len-1] = '\0'; strLen = (int)XSTRLEN(dp);
-    len -= (int)strLen; dp += strLen;
+    len -= (unsigned char)strLen; dp += strLen;
```

Comment:
Would prefer to see the `len` variable changed from `unsigned char` to `int`. Is cipher description always < 255?